build-system/tasks/e2e/describes-e2e.js

```diff
@@ -258,6 +262,11 @@ class AmpPageFixture {
     if (initialRect) {
```

Comment:
```javascript
const {
  initialRect = DEFAULT_E2E_INITIAL_RECT,
} = this.spec;
```
and remove this `if`. Better remove branching flows and normalize things on the configuration side of things.

Reply:
wow that is valid syntax? that's cool